integration/embed_test.go

```diff
@@ -409,3 +301,216 @@ func embedTestHelper(ctx context.Context, client *api.Client, t *testing.T, req
 
 	return client.Embed(ctx, &req)
 }
+
+func TestEmbedTruncation(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Minute)
+	defer cancel()
+	client, _, cleanup := InitServerConnection(ctx, t)
+	defer cleanup()
+
```

Comment:
You can wrap this with
```
        for _, model := range libraryEmbedModels {
```

and change `all-minilm` below this to pick up each of the embed models.